### ISSUE TITLE
Plants dataset

### ISSUE THREAD

**Issue by bavla** [uci-ml-repo/ucimlrepo-feedback#34]
I came across the dataset Plants (https://archive-beta.ics.uci.edu/dataset/180/plants). When converting it into a two-mode network, I noticed that it contains an unknown state "gl" - is gl=dengl ? pe, I guess, is the Prince Edward Island?
The dataset description says Number of Instances: 22632, but there are 34781 lines!!??

vladimir.batagelj@fmf.uni-lj.si

**Comment by ap0nia** [comment]
Recorded in #40

**Comment by rlongjohn** [comment]
Thank you for your questions! The number of instances for this dataset has been corrected to 34,781. As for your questions about the location abbreviations, I think your guesses for the meaning of "gl" and "pe" seem reasonable. After a quick glance at a few examples in the [USDA Plants Database](https://plants.usda.gov/home), it appears they do use "gl" as an abbreviation for Greenland, and those plants with "pe" (at least the few examples I looked at) do seem to be found in Prince Edward Island.
